src/btree/bt_cursor.c

```diff
@@ -2223,13 +2236,13 @@ __wt_btcur_close(WT_CURSOR_BTREE *cbt, bool lowlevel)
 }
 
 /*
- * __wt_btcur_bounds_row_position --
+ * __wt_btcur_bounds_position --
  *     A unpositioned bounded cursor need to start its cursor next and prev walk from the lower or
```

Comment:
np: Not a part of this PR, but could you fix up the comment here? "An unpositioned bounded cursor needs to start its cursor next and prev walk..."